README.md

```diff
@@ -18,6 +18,8 @@ Your group will be both a "Customer" and a "Designer".  First you'll prepare for
 
 ***Q1.1 Briefly describe your Mission:***
 
+Enforcing positive behavior by discouraging bad habits. Example: social media blocker activated until application is sent to recruiter.
```

Comment:
Your example sticks out, as it is like a regular site blocker brought to an extreme. This is certainly intriguing as an IoT device would somehow be used to monitor specific tasks that the user must finish.